admin/tabs/general/GeneralTab.js

```diff
@@ -21,9 +21,14 @@ export const generalTab = hoistCmp.factory(
             tabs: [
                 {id: 'about', icon: Icon.info(), content: aboutPanel},
                 {id: 'config', icon: Icon.settings(), content: configPanel},
-                {id: 'users', icon: Icon.users(), content: userPanel},
+                {id: 'users', icon: Icon.users(), content: userPanel, omit: omitTab('hideUsersTab')},
                 {id: 'alertBanner', icon: Icon.bullhorn(), content: alertBannerPanel}
             ]
         }
     })
 );
+
+const omitTab = (field) => {
+    const conf = XH.getConf('xhAdminConsoleConfig', {});
```

Comment:
Can we call it `xhAdminAppConfig`?  Can we add a default value in hoist core, so easy for new app users to find it?   

Also this function should just be called `hideUsersTab()` doesn't seem like it is ever going to be a general tab hiding mechanism

Reply:
We discussed the merits of adding a bootstrapped value in hoist-core.  I voted against it for the moment, while we only have one little flag here, and a rarely used one at that. 

Don't feel strongly about it, but in general I don't mind the pattern of supporting some more rare edge-case configs with baked-in defaults and keeping the list in most apps a bit more easily scannable.

Agree re. `omitTab` - a pattern I've used in a few places is to have a getter on my *model* class like:

`get conf() {return XH.getConf('someJsonConfigForMyModel', {})}` 

and then in the component here you could go with `omit: model.conf.hideUsersTab`

Reply:
Woops - got caught up in another convo and submitted this too late - thanks for merging and getting this change in!